publishconf.py

```diff
@@ -32,3 +32,5 @@
 
 PIWIK_URL = 'stats.data.gouv.fr'
 PIWIK_SITE_ID = 60
+
+DISPLAY_PAGES_ON_MENU = False
```

Comment:
Seems to have broken the header: see https://getudata.org/

Reply:
Do you want to fix it or shall I give it a try?